services/pgnotifier/pgnotifier.go

```diff
@@ -355,6 +365,31 @@ func (notifier *PgNotifierT) Publish(jobs []JobPayload, priority int) (ch chan [
 		pkgLogger.Errorf("PgNotifier: Error publishing messages: %v", err)
 		return
 	}
+
+	uploadSchemaJSON, err := json.Marshal(struct {
+		UploadSchema whUtils.SchemaT
+	}{
+		UploadSchema: *schema,
+	})
+	if err != nil {
+		return
```

Comment:
Same as below to add some context when returning the error. Wdyt ?

Reply:
Done.